### PR TITLE
feat: Setup an Organization banner

### PR DESCRIPTION
**Pull request by leog**
## What does this PR do?

Implements the banner in Event Types to present organizations.

<kbd>image</kbd>

<kbd>image</kbd>

### CODE REVIEW

**Review by PeerRich — CHANGES_REQUESTED** [review]
@leog this component exists already, check out all the other "UpdateScreen" components used in insights

File: apps/web/pages/event-types/index.tsx [hunk 1]
```diff
@@ -731,6 +733,40 @@ const CreateFirstEventTypeView = () => {
...
+  return (
+    <div className="bg-inverted text-inverted relative mx-4 mt-4 h-56 max-w-full rounded-md bg-[url('/noise.svg')] md:mt-8 lg:mx-12">
+      <div className="h-full w-full rounded-md bg-[url('/grid.png')] bg-[length:60%_100%] bg-[100%_0rem] bg-no-repeat">
+        <div className="h-full gap-4 rounded-md bg-[url('/orgs_banner.png')] bg-[length:50%] bg-[120%_4.4rem] bg-no-repeat">
+          <Button
+            variant="icon"
+            StartIcon={X}
+            color="minimal"
+            className="hover:text-muted absolute top-0 right-0 text-white hover:bg-transparent"
+          />
```
Comment (PeerRich) [inline apps/web/pages/event-types/index.tsx:748]: this button has no onClick

**Comment by leog** [comment]
> @leog this component exists already, check out all the other "UpdateScreen" components used in insights

@PeerRich Sorry, didn't mean to create the PR without being a draft, working on the onClick right now. And we will change this banner not to be prominent per our sync today with @Jaibles so it might need to be its own banner different from what we have.

**Comment by PeerRich** [comment]
lets deprioritize this

we dont need an advertisement banner when the core feature isnt done yet

**Pull request closed without merging**